mondrian/src/main/java/mondrian/rolap/SqlStatement.java

```diff
@@ -506,7 +506,7 @@ protected Dialect createDialect() {
    */
   public ResultSet getWrappedResultSet() {
     return (ResultSet) Proxy.newProxyInstance(
-      null,
+      ResultSet.class.getClassLoader(),
```

Comment:
@peterrinehart This was reported in https://jira.pentaho.com/browse/MONDRIAN-2660
Please close that issue with a comment that it is fixed in this pull request.